x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/rules/utils.ts

```diff
@@ -206,3 +206,32 @@ export const transformOrImportError = (
     });
   }
 };
+
+export const getMapDuplicates = (
```

Comment:
Should we just use [`_.countBy`](https://lodash.com/docs/4.17.15#countBy) here instead?

Reply:
I think that could look good. I will take a stab at it.